src/main/java/io/jenkins/plugins/propelo/commons/service/LevelOpsPluginConfigService.java

```diff
@@ -48,6 +66,13 @@ public LevelOpsConfig load(String s) throws Exception {
                     }
                 });
     }
+
+    public ApplicationType getApplicationType(File pluginFile) throws IOException {
+        XmlFile xml = new XmlFile(pluginFile);
+        PropeloPluginImpl propeloPlugin = (PropeloPluginImpl) xml.read();
```

Comment:
This is error prone. The correct way to do this is.
PropeloPluginImpl.getInstance()
That way the file path does not need to be hardcoded.

Reply:
done